Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -303,7 +303,7 @@ void VertexShaderManager::SetConstants()
     // NOTE: If we ever emulate antialiasing, the sample locations set by
     // BP registers 0x01-0x04 need to be considered here.
     const float pixel_center_correction = 7.0f / 12.0f - 0.5f;
-    const bool bUseVertexRounding = g_ActiveConfig.bVertexRounding && g_ActiveConfig.iEFBScale != 1;
+    const bool bUseVertexRounding = g_ActiveConfig.UseVertexRounding();
```

Comment:
It seems like vertex rounding would now occur while running at native resolution or is the check incorporated into `UseVertexRounding()`?

Reply:
https://github.com/dolphin-emu/dolphin/blob/80771ec89e32b6c36d32a2c3d99b7cd19a3adf43/Source/Core/VideoCommon/VideoConfig.h#L249

Reply:
One minor comment.  In WXWidget version, we'd disable the checkbox if it was native to reinforce it wasn't applicable.  That got lost when we moved to Qt.